samples/extensions/vertex_dynamic_state/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2022, Mobica Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 the "License";
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+ 
+get_filename_component(FOLDER_NAME ${CMAKE_CURRENT_LIST_DIR} NAME)
+get_filename_component(PARENT_DIR ${CMAKE_CURRENT_LIST_DIR} PATH)
+get_filename_component(CATEGORY_NAME ${PARENT_DIR} NAME)
+add_definitions(-DVK_ENABLE_BETA_EXTENSIONS)
+#add_definitions(-DDEBUG_SAMPLE)
+
+
+
+
+add_sample(
+    ID ${FOLDER_NAME}
+    CATEGORY ${CATEGORY_NAME}
+    AUTHOR "Mobica"
+    NAME "vertex_dynamic_state"
```

Comment:
This name is displayed on the mobile launcher, so it should be the human readable name "Vertex Dynamic State" instead.

Reply:
Thanks for explanation of this NAME variable. I have implemented this change.